source/fundamentals/authentication/mechanisms.txt

```diff
@@ -201,6 +181,51 @@ The driver checks for your credentials in the following sources in order:
       .. literalinclude:: /code-snippets/authentication/aws.js
          :language: javascript
 
+   .. tab:: Environment Variables
+      :tabid: environment variables
+
+      To authenticate to your MongoDB instance using AWS credentials stored in
+      environment variables, you must specify ``MONGODB-AWS`` as the
+      value of the ``authMechanism`` parameter in your connection string.
+      Then, set the following environment variables by using
+      ``bash`` or a similar shell as shown in the following example:
+
+      .. code-block:: bash
+
+         export AWS_ACCESS_KEY_ID=<awsKeyId>
+         export AWS_SECRET_ACCESS_KEY=<awsSecretKey>
+         export AWS_SESSION_TOKEN=<awsSessionToken>
+
+      .. note::
+
+         Omit the line containing ``AWS_SESSION_TOKEN`` if you don't need an AWS
+         session token for that role.
+      
+      After you've set the preceding environment variables, specify the ``MONGODB-AWS``
+      authentication mechanism in your connection string as shown in the following example:
```

Comment:
This sentence seems to conflict with what's said in the intro paragraph on this tab. In the intro it states to specify ``MONGODB-AWS`` in the connection string, _then_ to set the environment variables. But here it says to specify it in the connection string _after_ you've set the variables.

I think a slight rewording of either this sentence or the earlier paragraph would help. Maybe either removing the "Then, " in the first paragraph, or by rewording this paragraph to simply introduce what's happening in the following example without being specific on which order it should be done in. 

Happy to workshop more if you want